frontend/src/app/shared/components/comment/comment.component.scss

```diff
@@ -0,0 +1,75 @@
+@import 'variables';
+@import 'colors';
+@import 'mixins';
+
+.comment {
+  width: 570px;
+  height: 170px;
```

Comment:
nu cred ca trebuie acest height, ar trebui sa aiba niste copii ceva padding daca e cazul, iar containerul sa fie inalt in functie de continut. ( ce are copilul )

Reply:
fixed